### PR TITLE
Make final var  when is possible.

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: src/main/org/apache/tools/ant/taskdefs/AbstractJarSignerTask.java [hunk 1]
```diff
@@ -381,7 +381,7 @@ protected void setCommonOptions(final ExecTask cmd) {
      * @throws BuildException if the property is not correctly defined.
      */
     protected void declareSysProperty(
-        ExecTask cmd, Environment.Variable property) throws BuildException {
+            final ExecTask cmd, final Environment.Variable property) throws BuildException {
```
Comment (bodewig) [inline src/main/org/apache/tools/ant/taskdefs/AbstractJarSignerTask.java:384]: why did you change indentation here?

**Comment by bodewig** [comment]
This is a huge change when it comes to the number of lines touched. When faced with such a big change I always fear I may overlook something subtle.

Personally I don't believe making the argument of a one-line method that just calls another method passing on the argument is improving anything. YMMV.

**Comment by arturobernalg** [comment]
> This is a huge change when it comes to the number of lines touched. When faced with such a big change I always fear I may overlook something subtle.
> 
> Personally I don't believe making the argument of a one-line method that just calls another method passing on the argument is improving anything. YMMV.

HI @bodewig 
I close the PR
TY

**Comment by bodewig** [comment]
@arturobernalg I'm by far not the only Ant committer, so please don't feel forced to close a PR just because I comment in an unfavorable way.

**Comment by arturobernalg** [comment]
> @arturobernalg I'm by far not the only Ant committer, so please don't feel forced to close a PR just because I comment in an unfavorable way.

HI @bodewig 
TY for the feedback. I'll reopen the PR.
TY
